site/Testing/index.html

```diff
@@ -162,12 +181,596 @@ <h2 id="list-create-edit-delete-or-history-an-asset">List, create, edit, delete
 <p>The <strong>edit</strong> screen is accessed by selecting the <strong>edit icon</strong> in an asset's list window.</p>
 <p>The <strong>removal</strong> of an asset is requested by selecting the <strong>delete icon</strong> in the asset's list window.</p>
 <p>The <strong>history</strong> of an asset (all changes recorded in the ledger) can be viewed by selecting the <strong>history icon</strong> in an asset's window.</p>
-<h2 id="executing-an-transaction">Executing an transaction</h2>
+<h3 id="executing-an-transaction">Executing an transaction</h3>
 <p>The <strong>transaction</strong> execution can be performed by selecting the transaction in the sidebar.</p>
 <p>For example, for transaction <strong>UpdateBookTenant</strong></p>
 <p><img alt="Transaction" src="../img/transaction-cc-webclient.png" /></p>
-              
-            </div>
+
+<h2 id="using-godog">Using Godog</h2>
```

Comment:
@mikaellafs Did you added the GoDog content just here on the `site` folder? I'm not finding it on the `docs` MDs

Reply:
uhh, you're right, I forgot :grimacing: I'll do that.